src/styles/index.scss

```diff
@@ -8,6 +8,10 @@ body {
   height: 100%;
 }
 
+div[id="root"] {
```

Comment:
May I get the context behind this change?

Reply:
our `root` div (where react injects the app) is not giving 100% height oddly enough, which _might_ have led to overflow on workspace

i think i fixed the overflow issue downstream so i'm ok with removing this